internal/service/resiliencehub/testdata/tmpl/app_tags.gtpl

```diff
@@ -0,0 +1,30 @@
+resource "aws_resiliencehub_app" "test" {
```

Comment:
Typically, the configuration for the tagging test should be as simple as possible, often only using required attributes.